gamification_app/views.py

```
@@ -37,6 +40,9 @@ def get_user_score(request,username):
     
     except User.DoesNotExist:
         return Response({'message': 'User not found'} ,status=404)
+    except Exception as ex:
+        logger.exception(ex)
+        return Response(status=500, data={'message': 'Some error occurred while getting user score.'})
```

Or, you can raise the exception without having to return a 500 error. However, that's optional.